### ISSUE TITLE
Неверный формат словаря

### ISSUE THREAD

**Issue by Create789** [Koziev/GrammarEngine#18]
Здравствуйте, пытаюсь запустить прекомпиллед парсер из  GrammarEngine-master (под win64)

```
C:\Users\Anton\Downloads\!new_dwnl\GrammarEngine-master\GrammarEngine-master\src
\scripts\Parser>..\..\exe64\Parser.exe -verbose -tagger 0 -parser 0 -fuzzy_wordr
ecog 1 -lemmatizer 0 -eol -d ..\..\bin-windows64\dictionary.xml 1.txt -o 1.txt.p
arsing.xml
Loading dictionary from [C:\Users\Anton\Downloads\!new_dwnl\GrammarEngine-master
\GrammarEngine-master\src\bin-windows64\dictionary.xml]...

Ошибка: LEM: Assertion failed. Condition: Dictionary file has unknown or incompa
tible formatfile: e:\mvoice\lem\ai\some\dictionary.cpp line: 1931
Errcode=0 text=Операция успешно завершена.

Error:
Could not load the dictionary using [C:\Users\Anton\Downloads\!new_dwnl\GrammarE
ngine-master\GrammarEngine-master\src\bin-windows64\dictionary.xml]
```

Та же ошибка LEM в убунте (копилил из исходников парсер)

**Comment by Create789** [comment]
И еще вопрос в режиме Parser_RestHttpService.exe флаг -emit_morph 1 не работает? по разному пробовал - 
![изображение](https://user-images.githubusercontent.com/37928875/61588469-c59e8f80-aba4-11e9-92de-a3b6a2dcff42.png)

**Comment by Koziev** [comment]
Привет,

> Та же ошибка LEM в убунте (копилил из исходников парсер)
> Подскажите что делаю не так? Возможно парсер надо компилить под ВИН тоже? Из коробки не запускается?
> 

Сейчас проверил: скачал проект на отдельную вин машину, запустил в папке scripts\Parser файл Parse.cmd

Парсер запустился, словарь загрузил, входной текстовый файл разобрал нормально.

То есть из коробки парсер и словарь должны нормально работать без самостоятельной сборки.

Может быть, Вы что-то по-другому делаете?

**Comment by Create789** [comment]
Да, все ок, спасибо. Файлы не выкачались до конца, поспешил видимо.
А в Parser_RestHttpService.exe  -emit_morph 1 может быть сделаете?
